modules/criteoBidAdapter.js

```diff
@@ -427,6 +427,7 @@ function buildCdbRequest(context, bidRequests, bidderRequest) {
   let schain;
   let userIdAsEids;
   const request = {
+    id: bidderRequest.ortb2?.source?.tid || generateUUID(),
```

Comment:
this looks a bit like openrtb, $.id should be unique to the request and can be  generateUUID() or bidderRequestId